documentation/faq/80-integrations.md

```diff
@@ -47,6 +47,19 @@ You can also run code in `onMount` if you'd like to run it after the component h
 </script>
 ```
 
+If you prefer to use static imports instead, check out [vite-plugin-iso-import](https://github.com/bluwy/vite-plugin-iso-import) to support the `?client` import suffix. The import will be stripped out in SSR builds.
```

Comment:
the existing text is outdated. I believe you should actually be able to use static imports for the most part today. I sent a PR to update that and several other things in https://github.com/sveltejs/kit/pull/2112

Reply:
Interesting, so in regards to the new docs from that PR, copied here:

```html
import { onMount } from 'svelte';
import { method } from 'some-browser-only-library');

onMount(async () => {
	method('hello world');
});
```

I tested this on the [rollup repl](https://rollupjs.org/repl/?version=2.56.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMG1ldGhvZCUyMCU3RCUyMGZyb20lMjAnLiUyRmxpYnJhcnknJTVDbiU1Q25mdW5jdGlvbiUyMG9uTW91bnQoZm4pJTIwJTdCJTdEJTVDbiUyRiUyRiUyMGZ1bmN0aW9uJTIwb25Nb3VudChmbiklMjAlN0IlMjBmbigpJTIwJTdEJTVDbiU1Q25vbk1vdW50KCgpJTIwJTNEJTNFJTIwJTdCJTVDbiUyMCUyMG1ldGhvZCgpJTVDbiU3RCklNUNuJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlN0QlMkMlN0IlMjJuYW1lJTIyJTNBJTIybGlicmFyeS5qcyUyMiUyQyUyMmNvZGUlMjIlM0ElMjJkb2N1bWVudC50aXRsZSUyMCUzRCUyMCdBY2Nlc3MlMjB0aGUlMjBicm93c2VyJTIwQVBJcyUyMG9uJTIwaW1wb3J0JyU1Q24lNUNuZXhwb3J0JTIwZnVuY3Rpb24lMjBtZXRob2QoKSUyMCU3QiU1Q24lNUN0Y29uc29sZS5sb2coJ2h1cnJhaCcpJTVDbiU3RCUyMiU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE), and wouldn't that still preserve side-effects of the browser-only library? It still needs dynamic imports in order to avoid the side effects ([another repl](https://rollupjs.org/repl/?version=2.56.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmZ1bmN0aW9uJTIwb25Nb3VudChmbiklMjAlN0IlN0QlNUNuJTJGJTJGJTIwZnVuY3Rpb24lMjBvbk1vdW50KGZuKSUyMCU3QiUyMGZuKCklMjAlN0QlNUNuJTVDbm9uTW91bnQoYXN5bmMlMjAoKSUyMCUzRCUzRSUyMCU3QiU1Q24lNUN0Y29uc3QlMjAlN0IlMjBtZXRob2QlMjAlN0QlMjAlM0QlMjBhd2FpdCUyMGltcG9ydCUyMCgnLiUyRmxpYnJhcnknKSU1Q24lMjAlMjBtZXRob2QoKSU1Q24lN0QpJTVDbiUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmxpYnJhcnkuanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyZG9jdW1lbnQudGl0bGUlMjAlM0QlMjAnQWNjZXNzJTIwdGhlJTIwYnJvd3NlciUyMEFQSXMlMjBvbiUyMGltcG9ydCclNUNuJTVDbmV4cG9ydCUyMGZ1bmN0aW9uJTIwbWV0aG9kKCklMjAlN0IlNUNuJTVDdGNvbnNvbGUubG9nKCdodXJyYWgnKSU1Q24lN0QlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyZXMlMjIlMkMlMjJuYW1lJTIyJTNBJTIybXlCdW5kbGUlMjIlMkMlMjJhbWQlMjIlM0ElN0IlMjJpZCUyMiUzQSUyMiUyMiU3RCUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlN0QlMkMlMjJleGFtcGxlJTIyJTNBbnVsbCU3RA==))

Reply:
That's why I started the sentence with "If the library you'd like to use is side-effect free" :wink:

Reply:
Ooops. Totally missed that part! I'll update this paragraph then to continue on from the updated stuff in #2112

Reply:
I've updated the docs to anticipate the change in #2112, so when that's merged we can merge this as well (and hopefully no conflicts). The new paragraph should be appended last in the section.